cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/CareGaps.java

```diff
@@ -174,28 +168,18 @@ public CareGaps.When periodStart(String periodStart) {
             return this;
         }
 
-        public CareGaps.When practitioner(String practitioner) {
-            this.practitioner = practitioner;
-            return this;
-        }
-
         public CareGaps.When subject(String subject) {
             this.subject = subject;
             return this;
         }
 
-        public CareGaps.When organization(String organization) {
-            this.organization = organization;
-            return this;
-        }
-
         public CareGaps.When statuses(String statuses) {
             this.statuses.add(statuses);
             return this;
         }
 
         public CareGaps.When measureIds(String measureIds) {
-            this.measureIds.add(measureIds);
+            this.measureIds.add(new IdType(measureIds));
```

Comment:
Suggestion:  use the constructor with the resource type.  InMemoryRepository doesn't necessarily populate the resource type on IDs and this can sometimes cause errors.